### PR TITLE
Access to values in SpdzSInt and FieldElement. Closable on SecureComputationEngine.

### PR DESCRIPTION
**Pull request by eoasecata**

### CODE REVIEW

**Comment by GuutBoy** [comment]
Come on! Could you please comment your PR's a bit more? It's really hard to follow what is going on this way 😛

**Comment by jonas-lj** [comment]
Sure! We did all communication about this PR through other channels, but I see why that doesn't really help you :). Partisia requested some convenience methods to make it easier to extract shares from secret shared values, eg. extracting BigIntegers. They also needed SecureComputationEngine and Mascot to be Closable -- my guess is because they run multiple instances of these and not just a single one which starts and stops with the application as in the existing demos.
